bitcoin/src/bip32.rs

```diff
@@ -30,12 +30,12 @@ const VERSION_BYTES_TESTNETS_PUBLIC: [u8; 4] = [0x04, 0x35, 0x87, 0xCF];
 const VERSION_BYTES_TESTNETS_PRIVATE: [u8; 4] = [0x04, 0x35, 0x83, 0x94];
 
 /// The old name for xpub, extended public key.
-#[deprecated(since = "0.31.0", note = "use xpub instead")]
-pub type ExtendendPubKey = Xpub;
```

Comment:
Again, these must not be removed now that they are released.

Reply:
Oh woops, that is what I get for doing PRs in the middle of the night, will re-do the changes manually instead of by using `git cherry-pick`.